packages/webapp/src/components/Animals/AddAnimalsForm/MoreAnimalsCard.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import AddAnimalsCard from './AddAnimalsCard';
+import styles from './styles.module.scss';
+import { useTranslation } from 'react-i18next';
+import clsx, { ClassValue } from 'clsx';
```

Comment:
```suggestion
import { ClassValue } from 'clsx';
```

The `className` prop isn't actually used by the component. Not sure if it's an oversight and we should build it in (apply it somewhere), or delete it entirely... probably delete, right? 😅